src/routes/account/plan/+page.svelte

```diff
@@ -5,7 +5,7 @@
 
   const redirectParam = $page.url.searchParams.get('successRedirect');
   const selectBasic = async () => {
-    if (redirectParam && redirectParam === ('app' || 'previewApp')) {
+    if (redirectParam && (redirectParam === 'app' || redirectParam === 'previewApp')) {
```

Comment:
This was a bug, `redirectParam === ('app' || 'previewApp'))` always evaluates as `redirectParam === 'app'`